src/complex/Utilities/Parsing/DREAM3D/Dream3dIO.cpp

```diff
@@ -589,18 +621,14 @@ Result<complex::DataStructure> complex::DREAM3D::ImportDataStructureFromFile(con
   }
 
   H5::ErrorType error = 0;
-  DataStructure dataStructure = ImportDataStructureFromFile(fileReader, error);
-
-  return {std::move(dataStructure)};
+  return ImportDataStructureFromFile(fileReader, error);
 }
 
-complex::Pipeline complex::DREAM3D::ImportPipelineFromFile(const H5::FileReader& fileReader, H5::ErrorType& errorCode)
+Result<complex::Pipeline> complex::DREAM3D::ImportPipelineFromFile(const H5::FileReader& fileReader)
 {
-  errorCode = 0;
-
   if(GetPipelineVersion(fileReader) != k_CurrentPipelineVersion)
   {
-    return {};
+    return MakeErrorResult<Pipeline>(k_InvalidPipelineVersion, fmt::format("Could not parse Pipeline version {}. Expected version: {}", GetPipelineVersion(fileReader), k_CurrentFileVersion));
   }
 
   auto pipelineGroupReader = fileReader.openGroup(k_PipelineJsonTag);
```

Comment:
On line 636, `MakeErrorResult` is needed.

Reply:
fixed in  e44f57552163731f5aa63b5deaf94227938a64c0